### ISSUE TITLE
Training from scratch problem

### ISSUE THREAD

**Issue by TopGun666** [JiawangBian/SC-SfMLearner-Release#47]
Hi Jiawang,

Thanks for your effort in the updated version. I found some problems when I tried to train from scratch with default hyper-parameters. It looks overfitted after 3 epochs. Another question is that the GC loss looks conflicted with photometric loss at first. 

![loss_summary](https://user-images.githubusercontent.com/23215825/84782765-0011e600-b01b-11ea-8137-33d59ee5878d.png)
![each_loss](https://user-images.githubusercontent.com/23215825/84782786-0607c700-b01b-11ea-94f4-a7818caf10e4.png)

**Comment by JiawangBian** [comment]
1. Overfitted on KITTI? I tried many times but didn't meet this issue. I may need more info to reproduce.
2. Yes. GC loss and smoothness loss need some time to converge to correct ranges.

**Comment by TopGun666** [comment]
Yes, I trained it on KITTI.

**Comment by JiawangBian** [comment]
Do you use GT for validation? What's the metric for the training loss and validation loss in the first figure? Is it AbsRel or photometric+GC+smooth?

**Comment by TopGun666** [comment]
I run the default `train_resnet50_pose_256.sh`. There is no `--with-gt` in the default scripts. Fig1 is the curve of the log `progress_log_summary_csv`.

**Comment by JiawangBian** [comment]
I see. I didn't provide Gt for KITTI Odometry dataset. Actually, the unsupervised loss (photometric+GC+smoothness) is not good enough for validation purpose. You just need train the model for more than about 50 epoches. Then the 'best' or the 'latest' models are both good enough.

**Comment by TopGun666** [comment]
Thanks for your reply. Did you change the learning rate during training?

**Comment by JiawangBian** [comment]
Never. I find that it doesn't matter. However, I suggest training from ImageNet pretrained model. It would be much better, and it is the main reason why I update network in this version. ResNet pretrained models are publicly available. So why not use.

**Comment by TopGun666** [comment]
Thanks a lot. I'll try.
